src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -180,18 +180,18 @@ public String getVersion() {
 
     private Network nodeBreakerNetwork;
 
-    private static void assertResultsEquals(ShortCircuitAnalysisResult result, org.gridsuite.shortcircuit.server.dto.ShortCircuitAnalysisResult resultDto) {
-        assertEquals(result.getFaultResults().size(), resultDto.getFaults().size());
-        List<FaultResult> orderedFaultResults = result.getFaultResults().stream().sorted(Comparator.comparing(fr -> fr.getFault().getId())).collect(Collectors.toList());
-        List<org.gridsuite.shortcircuit.server.dto.FaultResult> orderedFaultResultsDto = resultDto.getFaults().stream().sorted(Comparator.comparing(fr -> fr.getFault().getId())).collect(Collectors.toList());
+    private static void assertFaultResultsEquals(ShortCircuitAnalysisResult result, List<org.gridsuite.shortcircuit.server.dto.FaultResult> faultResults) {
+        assertEquals(result.getFaultResults().size(), faultResults.size());
+
+        // When we do not provide sort parameter to the request, order is not guaranteed, so we sort results here to not test their order
+        List<FaultResult> orderedFaultResults = result.getFaultResults().stream().sorted(Comparator.comparing(fr -> fr.getFault().getId())).toList();
```

Comment:
assertFaultResultsEquals() and assertSortedFaultResultsEquals() should be homogeneous. 
Wether you assume that the results given as "ShortCircuitAnalysisResult result" are sorted and you don't need to sort them. Wether you assume they are not sorted and you should sort them in both methods